packages/ui-toolkit/stories/IconStore.stories.tsx

```diff
@@ -67,3 +68,28 @@ export const AllMaterialIcons = () => {
     </div>
   );
 };
+
+
+export const AllMintIcons = () => {
+  return (
+    <div className="story_icon_all_image_list">
+      {
+        Object.keys(mintIcons).map(icon => {
+          const IconComponent = mintIcons[icon];
+          const importCode = `import { ${icon} } from '@groww-tech/icon-store/mi';`;
```

Comment:
shouldn't it be `@groww-tech/icon-store/mint-icons` ?

Reply:
corrected